src/components/Output/subcomponents/SummaryWidget.jsx

```diff
@@ -34,7 +34,7 @@ const cardContentStyles = {
 
 /// /// /// COMPONENTS /// /// ///
 const CustomWidthTooltip = styled(({ className, ...props }) => (
-  <Tooltip {...props} classes={{ popper: className }} />
+  <PSATooltip {...props} classes={{ popper: className }} />
```

Comment:
Please put the `<CustomTypography>` on line 66 into `tooltipContent` of `<CustomWidthTooltip>`